task.sh

```diff
@@ -1,43 +1,84 @@
 #!/bin/bash
+echo "1) Running script."
 
+echo "2) Creating '/task' directory with subdirectories."
 # создаём каталог task с вложенными директориями
 # task
 #   dir1
 #   dir2
 #   dir3
 #       dir4
-
+for i in {1..3}
+    do
+	if  [[ $i -eq 3 ]]; then
+	    mkdir -p ./task/dir$i/dir$((i+1))
+	else
+	    mkdir -p ./task/dir$i
+	fi
+    done
+
+echo "3) Changing current directory."
 # изменяем текущую директорию на task
+cd ./task
```

Comment:
Везде можно убрать `./`, это некритично, но просто зачем
```suggestion
cd task
```